### DISCUSSION TITLE
PHP 8.1 compatibility

### DISCUSSION THREAD

**Discussion by jakublabno** [php-amqplib/php-amqplib#1007] [Q&A]
Are you planning to add PHP 8.1 compatibility soon?

**Comment by andrew-demb** [comment]
The package is allowed to be installed on PHP 8.1, isn't it?
https://github.com/php-amqplib/php-amqplib/blob/0bec5b392428e0ac3b3f34fbc4e02d706995833e/composer.json#L32
